assets/app/vue/views/MailView/sections/EmailSettingsSection/components/EmailAliasForm.vue

```diff
@@ -67,7 +67,11 @@ const onSubmit = () => {
 
 <template>
   <template v-if="step === EMAIL_ALIAS_STEP.INITIAL">
-    <primary-button variant="outline" @click="step = EMAIL_ALIAS_STEP.SUBMIT">
+    <primary-button
+      variant="outline"
+      @click="step = EMAIL_ALIAS_STEP.SUBMIT"
+      :disabled="allowedDomains.length === 0"
+    >
```

Comment:
Sorry can we add a disabled tooltip mentioning they have to add a custom domain before they can add an alias?